portal_blocks_documentation/docs.py

```diff
@@ -42,7 +45,12 @@ def ensure_index_md(block_name: str):
             "draft: false",
             f"geekdocFilePath: portal_blocks/{block_name}/_index.md",
             "---",
-            f"# {block_name}"
+            f"# {block_name}",
+            '{{< include file="' + MOUNT_DIR + '/' + block_name + '/docs/official.md" >}}',
+            "",
+            f"# Community Additions",
+            "",
+            '{{< include file="' + MOUNT_DIR + '/' + block_name + '/docs/extra.md" >}}'
```

Comment:
We cannot use include shortcode as hugo server runs has render caching for each page, only the final page is cached and if a dependency of a child page is changed, the places where it is used are not updated.

for example, say we take the block `Abort` and add/change stuff in its `docs/extra.md` the cache is only burst for the extra.md file that `abort.md` will start the same. 

the only thing that is not cached is direct layouts `layout/blocks`, and we include the content on the docs there so that we can have live update of the changes

Reply:
i should have documented this my bad